SPECS-EXTENDED/liboggz/liboggz.spec

```diff
@@ -0,0 +1,217 @@
+Summary:        Simple programming interface for Ogg files and streams
+Name:           liboggz
+Version:        1.1.1
+Release:        20%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.xiph.org/oggz/
+Source0:        https://downloads.xiph.org/releases/liboggz/%{name}-%{version}.tar.gz
+# Always have oggz_off_t == loff_t even on 64-bit platforms
+Patch0:         liboggz-1.1.1-multilib.patch
+BuildRequires:  doxygen
+BuildRequires:  gcc
+BuildRequires:  libogg-devel >= 1.0
+
+%description
+Oggz provides a simple programming interface for reading and writing
+Ogg files and streams. Ogg is an interleaving data container developed
+by Monty at Xiph.Org, originally to support the Ogg Vorbis audio
+format.
+
+%package devel
+Summary:        Files needed for development using liboggz
+Requires:       libogg-devel >= 1.0
+Requires:       liboggz = %{version}-%{release}
+Requires:       pkgconfig
+
+%description devel
+Oggz provides a simple programming interface for reading and writing
+Ogg files and streams. Ogg is an interleaving data container developed
+by Monty at Xiph.Org, originally to support the Ogg Vorbis audio
+format.
+
+This package contains the header files and documentation needed for
+development using liboggz.
+
+%package doc
+Summary:        Documentation for liboggz
+Requires:       liboggz = %{version}-%{release}
+
+%description doc
+Oggz provides a simple programming interface for reading and writing
+Ogg files and streams. Ogg is an interleaving data container developed
+by Monty at Xiph.Org, originally to support the Ogg Vorbis audio
+format.
+
+This package contains HTML documentation needed for development using
+liboggz.
+
+%prep
+%setup -q -n %{name}-%{version}
```

Comment:
Consider using %autosetup

Reply:
it seems the `%patch -b .multilib` is needed for the build.